matching-service/src/controllers/matchingController.ts

```diff
@@ -0,0 +1,223 @@
+import { User, hasCommonDifficulties } from "../model/user";
+import { Kafka } from "kafkajs";
+import queueManagerInstance from "../model/queueManager";
+import { on } from "events";
+
+const kafka = new Kafka({
+    clientId: 'matching-service',
+    brokers: ['localhost:9092'], // TODO: add to env variables
```

Comment:
with this, it seems likely that the matching service won't really match up between computers now right

it's fine since CS3219 doesnt expect this to work between computers full on

but just wondering how kafka would work here lol

Reply:
yup, if u want it to work between computers we need an actual ip address haha